CuratedContent/Jupyter4Science.md

```diff
@@ -0,0 +1,48 @@
+# Jupyter4Science: Better Practices for using Jupyter Notebooks for Science
+
+<!--deck text start-->
+[Jupyter4Science](https://jupyter4science.github.io/) is a **knowledge base** of original content and curated resources about developing and sharing Jupyter notebooks as they are used in the context of scientific applications.
+<!--deck text end-->
+
+#### Contributed by [Nicole Brewer](https://github.com/nicole-brewer)
+
+#### Publication date: March 31, 2023
+
+Resource information | Details 
+:--- | :--- 
+Resource name | Accessible Data and Code
+Website  | [https://jupyter4.science](https://jupyter4science.github.io/)
```

Comment:
Nicole, https://jupyter4.science doesn't work for me.  Should it?  If so, which URL would you prefer to advertise?  https://jupyter4.science or https://jupyter4science.github.io/? I think we should settle on one URL.

Reply:
@nicole-brewer Tagging you

Reply:
Thanks all. I've got the https://jupyter4.science URL working and have updated the page to reflect that

Reply:
Thanks!